EDU-597 Add parent display name

ahelmberger opened the pull request:
Closes https://educandu.atlassian.net/browse/EDU-597

adrianaluchian commented on src/services/storage-service.js:
```diff
@@ -134,27 +134,65 @@ export default class StorageService {
 
-    return mappedObjects;
+    return { parentDirectory, currentDirectory, objects };
   }
```
This method is decomposing and recomposing paths alot, feels like a test would be helpful here 🤔

ahelmberger replied:
True, it's the one function in this service that has no tests, I will add some.

ahelmberger replied:
Added, thanks for the suggestion, I actually found some edge case problems.

adrianaluchian commented:
The upper level navigation actually looks nice in the list view, we'll just need a better icon so it's good for the grid view as well

ahelmberger commented:
> The upper level navigation actually looks nice in the list view, we'll just need a better icon so it's good for the grid view as well.
> 
> How would this look like to you? <img alt="Screen Shot 2022-06-08 at 13 50 44" width="523" src="https://user-images.githubusercontent.com/8189923/172609499-714dc3c7-557e-46cc-88d1-553b5bf265c1.png">

Agree, that looks a lot nicer. Did you create this as an SVG or just manipulate the pixels (i.e. can you provide me yours or should a re-create it :smiling_imp:)?

adrianaluchian commented:
> > The upper level navigation actually looks nice in the list view, we'll just need a better icon so it's good for the grid view as well.
> > How would this look like to you? <img alt="Screen Shot 2022-06-08 at 13 50 44" width="523" src="https://user-images.githubusercontent.com/8189923/172609499-714dc3c7-557e-46cc-88d1-553b5bf265c1.png">
> 
> Agree, that looks a lot nicer. Did you create this as an SVG or just manipulate the pixels (i.e. can you provide me yours or should a re-create it 😈)?

Sure, as you're ok with it I've committed it to the assets repo, along with a version for the files list view (haven't checked how that one looks but feel free to use, adjust or delete it).

ahelmberger commented:
> > [...]
> > Agree, that looks a lot nicer. Did you create this as an SVG or just manipulate the pixels (i.e. can you provide me yours or should a re-create it smiling_imp)?
> 
> Sure, as you're ok with it I've committed it to the assets repo, along with a version for the files list view (haven't checked how that one looks but feel free to use, adjust or delete it).

Thanks so much :pray:

ahelmberger commented:
> > [...]
> > Agree, that looks a lot nicer. Did you create this as an SVG or just manipulate the pixels (i.e. can you provide me yours or should a re-create it smiling_imp)?
> 
> Sure, as you're ok with it I've committed it to the assets repo, along with a version for the files list view (haven't checked how that one looks but feel free to use, adjust or delete it).

Added and adjusted the new icons :+1:
